targets/standalone/public/index.html

```diff
@@ -1,65 +0,0 @@
-<!DOCTYPE html>
```

Comment:
Why did we remove this file?

Reply:
It was removed by mistake when updating its path, it is added back by the next PR https://github.com/Automattic/happychat-client/pull/180/files

Reply:
Ah, I see. You realize that if you add the file back in a different PR, you'll get the whole credit/blame? 😛

Reply:
Haha, I accidentally removed and added it back :) The file was moved from `standalone/public/index.html` to `standalone/index.html`. While commiting the removal here I used `git commit -am` which did not add the new file. In the next PR I used `git add --all` which added back the file that was on my machine (end of day PR splitting is fun)

Reply:
Hehe. :) So do you say that the file will keep its git history? If not, could you just re-add it back in this PR?

Reply:
Not quite sure what you mean, after all sub branches are merged and everything will be squashed into master they all will be under one commit.

Reply:
It confuses me a bit that we delete and add files unrelated to the PRs goal, so I also prefer we add it back. But! I'm new to this mega-PR building trick, so I may be overly cautious.

Reply:
I agree this was removed by mistake and I accidentally added it in another PR when splitting into smaller PRs.